packages/big-design/src/components/Dropdown/Dropdown.tsx

```diff
@@ -99,28 +125,28 @@ export const Dropdown = memo(
       strategy: positionFixed ? 'fixed' : 'absolute',
     });
 
-    const renderToggle = useMemo(() => {
-      return (
-        isValidElement(toggle) &&
-        cloneElement(toggle, {
-          ...getToggleButtonProps({
-            'aria-expanded': isOpen, // Because of memoization, we need to manually set this option
-            disabled,
-            ref: referenceRef,
-          }),
-        })
-      );
-    }, [disabled, getToggleButtonProps, isOpen, toggle]);
+    const clonedToggle =
+      isValidElement(toggle) &&
+      cloneElement(toggle, {
+        ...getToggleButtonProps({
+          'aria-haspopup': 'menu',
+          // Downshift sets this to a label id that doesn't exist
+          'aria-labelledby': undefined,
+          disabled,
+          ref: referenceRef,
+          role: 'button',
+        }),
+      });
```

Comment:
`aria-activedescendant` wasn't getting updated so I removed the memo. It's going to rerender any update anyways 🤷‍♂️